src/Devices/PHControl.cpp

```diff
@@ -67,7 +67,13 @@ PHControl::PHControl() {
       amplitude = EEPROM_TC::instance()->getPhSineAmplitude();
       sineStartTime = EEPROM_TC::instance()->getPhSineStartTime();
       break;
+    case FLAT_TYPE:
```

Comment:
I had understood from our discussion that these values were for debugging. What does it mean for a flat type to have a start and end time?